netebpfext/net_ebpf_ext_bind.c

```diff
@@ -54,22 +64,49 @@ ebpf_extension_data_t _net_ebpf_extension_bind_hook_provider_data = {
 const NPI_MODULEID DECLSPEC_SELECTANY _ebpf_bind_hook_provider_moduleid = {
     sizeof(NPI_MODULEID), MIT_GUID, {0xeab8f3d9, 0xab6c, 0x422e, {0x99, 0x4c, 0x7a, 0x80, 0x94, 0x3b, 0xc9, 0x20}}};
 
-const NPI_PROVIDER_CHARACTERISTICS _ebpf_bind_hook_provider_characteristics = {
-    0,
-    sizeof(NPI_PROVIDER_CHARACTERISTICS),
-    net_ebpf_extension_hook_provider_attach_client,
-    net_ebpf_extension_hook_provider_detach_client,
-    NULL,
-    {0,
-     sizeof(NPI_REGISTRATION_INSTANCE),
-     &EBPF_ATTACH_TYPE_BIND,
-     &_ebpf_bind_hook_provider_moduleid,
-     0,
-     &_net_ebpf_extension_bind_hook_provider_data},
-};
-
 static net_ebpf_extension_hook_provider_t* _ebpf_bind_hook_provider_context = NULL;
 
+//
+// Client attach/detach handler routines.
+//
+
+ebpf_result_t
+net_ebpf_extension_bind_on_client_attach(_In_ const net_ebpf_extension_hook_client_t* attaching_client)
```

Comment:
can this be static?  same question on line 100

Reply:
Fixed.